projects/observability/src/shared/services/metadata/handler/metadata-graphql-query-handler.service.ts

```diff
@@ -46,7 +46,8 @@ export class MetadataGraphQlQueryHandlerService
       onlySupportsAggregation: result.onlySupportsAggregation,
       onlySupportsGrouping: result.onlySupportsGrouping,
       allowedAggregations: result.supportedAggregations.flatMap(convertFromGraphQlMetricAggregationType),
-      groupable: result.groupable
+      groupable: result.groupable,
+      sortable: true // Assumption: All fields are sortable
```

Comment:
Why do we need to add this attribute? The rest of these are being served by the backend and can be configured (since they depend on things like data type), but sortability we don't need any extra data to determine.

Reply:
yes, you're right